### DISCUSSION TITLE
Green box widget for Android

### DISCUSSION THREAD

**Discussion by codad5** [community/community#63418] [Mobile]
It would be nice if android can have the GitHub activity green box widget, so that once can be able to add it to their holme screen .

Thanks

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @codad5! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've gone ahead and moved this to the correct category for you. 

To clarify your feedback request a bit, do you mean the [Contribution Graph](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#viewing-contributions-from-specific-times)?

**Comment by cehfisher** [comment]
Hi @codad5 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
